threescale_api/resources.py

```diff
@@ -726,6 +726,7 @@ def test_request(self, relpath=None, verify: bool = None):
 
         return client.get(relpath)
 
+
```

Comment:
this renaming doesn't seem to be relevant to the change in this PR, rather shouldn't be in this PR

Reply:
Fixed